_docs/hackathon-preperation-checklist.md

```diff
@@ -0,0 +1,43 @@
+# The Week Before
+- [x] testing task list
+
+- Make a team
+- Brainstorm ideas
+    - [link to project-ideas page?]
+- Update your software
```

Comment:
This is a bit vague; I'd nix it unless there are specific things that students should be updating or installing.
```suggestion
```